client/cmd/dexc/main.go

```diff
@@ -28,6 +28,14 @@ import (
 	"decred.org/dcrdex/dex"
 )
 
+// Register asset drivers.
+func init() {
+	bch.RegisterDriver()
+	btc.RegisterDriver()
+	dcr.RegisterDriver()
+	ltc.RegisterDriver()
+}
```

Comment:
Missing eth, which is registered on condition.

Reply:
Don't think eth is currently imported on master. Think it was removed sometime ago in preparation for the 0.3 release.

Having the asset drivers auto-registered whenever their respective packages are imported will cause issues for consumers that want to register a different driver for an asset, that's what motivated this change, making it the responsibility of consumers to call the `RegisterDriver` function as desired.

This PR opens the possibility for consumers to register a slightly different driver for dcr wallets.
For example , the default dcr driver that is registered above using `dcr.RegisterDriver()` has the following `Setup` instruction:
```
func (d *Driver) Setup(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network) (asset.Wallet, error) {
	return NewWallet(cfg, logger, network, &rpcWallet{})
}
```

With this PR, a different consumer might register a driver that does this instead:
```
func (d *Driver) Setup(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network) (asset.Wallet, error) {
	return NewWallet(cfg, logger, network, &spvWallet{})
}
```

If the default driver is auto-registered whenever the `client/asset/dcr` pkg is imported, consumers would be unable to register a different driver.

Reply:
I see! Thank you for the explanation. imo don't worry about eth, it's still in development.